kite-data/kite-data-core/pom.xml

```diff
@@ -271,5 +271,36 @@
         </plugins>
       </build>
     </profile>
+    <profile>
+      <id>hdp2-release</id>
```

Comment:
I can look at setting those by properties. We don't need the `install-hdp-classifier` execution of the install plugin during a release, we only do a deploy.